lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1086,27 +1086,38 @@ def restart_sctpd(self, wait_time=30):
                 "/usr/local/bin/config_stateless_agw.py sctpd_pre';"
                 "docker start sctpd mobilityd pipelined sessiond oai_mme",
             )
-        self.wait_for_restart_to_finish(wait_time)
 
-    @staticmethod
-    def wait_for_restart_to_finish(wait_time):
-        for j in range(wait_time):
-            print(
-                f"Waiting for {wait_time - j} seconds for restart to complete",
-            )
-            time.sleep(1)
+    def wait_for_restart_to_finish(self, wait_time):
+        """wait for started services to become active or until timeout
+
+        Args:
+            services: (list[str]) list of services to check
+            wait_time: (int) max time to wait for services to become active
+        """
+        if self._init_system == InitMode.DOCKER:
+            start_time = time.time()
+            all_services_active = False
+            while (
+                not all_services_active
+                and time.time() - start_time < wait_time
+            ):
+                all_services_active = self.check_if_magma_services_are_active()
+                time.sleep(1)
```

Comment:
Every second it will call `self.check_if_magma_services_are_active` and print logs for 21 services until they are active. I think we can increase the sleep duration to minimum 5 seconds. Considering it takes usually not less than 10 seconds to restart all services.

Reply:
That is not completely correct. The loop in `self.check_if_magma_services_are_active()` ends after the first inactive service, so the print output is not quite that extensive. Nevertheless, the sleep time was adapted.